From 5b0c65787cb2384ebb7624b93a1e796e13531e81 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Sat, 22 May 2021 19:50:52 +0300
Subject: [PATCH] [SW-43026] boot_info: increase max descriptors

Increase max allowed descriptors in boot_info from
5 to 32, as in preboot we can set more than 5 boot_info
images, otherwise storing in boot_info will fail.

Change-Id: I30135ab7b0718a3884a53b53a4dd355cf100da55
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/preboot/boot_info.c b/preboot/boot_info.c
index db903b9..1e20cec 100644
--- a/preboot/boot_info.c
+++ b/preboot/boot_info.c
@@ -14,7 +14,7 @@
 #include "boot_info.h"
 #include "uart.h"
 
-#define MAX_INFO_DESCS	5
+#define MAX_INFO_DESCS	32
 
 #define BOOT_INFO_SZ	(sizeof(struct hl_boot_info_ptrs) + \
 			sizeof(struct hl_boot_info_desc) * MAX_INFO_DESCS)
